Use existing test.key instead of dev.key for payment encryption

Rafikooo opened the pull request:
The `test.key` file already exists in the repository and is suitable for development/testing purposes. Using it eliminates the need for developers to generate a `dev.key` file locally, making the setup simpler and cleaner.

NoResponseMate commented on .gitignore:
```diff
@@ -41,6 +41,10 @@
 !/config/jwt/*-test.pem
 ###< lexik/jwt-authentication-bundle ###
 
+###> sylius/sylius ###
+/config/encryption/dev.key
+###< sylius/sylius ###
```
It would be better to change the value in `.env` to `/config/encryption/test.key` as it's already there by default and the innards of these files don't really matter as long as it's not prod.

Rafikooo committed:
Use existing test.key instead of dev.key for payment encryption

Changed SYLIUS_PAYMENT_ENCRYPTION_KEY_PATH to use test.key which already exists
in the repository, instead of dev.key which required developers to create it
locally. This eliminates the need to gitignore dev.key as it's no longer used.
